proposals/policies/async-decision.md

```diff
@@ -0,0 +1,48 @@
+# Web of Things (WoT) Asynchronous Decision Policy
+
+This is a proposal to make the review process for specification changes more efficient by using asynchronous tools, rather than following the current unwritten rule that pull requests can only be merged following a resolution in a synchronous weekly conference call.
+
+This proposal was originally [submitted](https://lists.w3.org/Archives/Public/public-wot-wg/2021Aug/0006.html) to the public-wot-wg mailing list on 3rd August 2021 and followed up by @benfrancis at https://github.com/w3c/wot/pull/1005/files.
+This version extends it by taking the reviews into account, while moving the problem and notes into an Appendix section.
+
+## Proposed Solution
+
+### Overall Idea
+
+It is proposed to use GitHub's built-in [code review tools](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) to formally review pull requests without having to wait for a decision in a synchronous weekly call.
+This form of asynchronous decision process specifies how to conduct asynchronous decisions that are already written in the Working Group's existing [Decision Policy](https://www.w3.org/2020/01/wot-wg-charter.html#decisions).
+
+### Non-normative Changes
+
+- For non-normative or editorial changes to the wording of a specification or its supporting documents, a code review approval from a single Editor of the specification is sufficient to merge a pull request.
+
+### Normative Changes
+
+- For normative or breaking changes to a specification, approval is required by at least two Editors of a specification that are not from the same organization. Editors should use GitHub's code review tool to either:
+  - Approve the pull request (approval may be conditional on making some small changes like fixing typos before landing, noted in a comment)
+  - Request changes to the pull request, explained in a comment
+  - Comment to say that they have no strong opinion or that they abstain, deferring to the opinion of other editors
+- A pull request can be landed once two editors have either provided their approval or deferred to the other editors.
```

Comment:
This sounds like if two editors defer the decision can be made.  Needs to be clarified.  Do you mean if one editor defers, a single editor can make the decision?   What if a document only has one editor?

Reply:
I would need @benfrancis opinion here but my understanding is that an editor can leave the responsability to another editor and that other editor should decide. We can simply change it to `A pull request can be landed once two editors have either provided their approval`

Regarding one editor: I think all our (REC track) documents should have at least two editors.

Reply:
I have removed this deferral text. No need to complicate things at this point. We can change a policy later on if needed.